.env.mainnet

```
@@ -1,46 +1,122 @@
 
-OP_NODE_BETA_EXTRA_NETWORKS=true
-OP_NODE_L2_ENGINE_AUTH=/tmp/engine-auth-jwt
-OP_NODE_L2_ENGINE_RPC=ws://geth:8551
-OP_NODE_LOG_LEVEL=info
-OP_NODE_METRICS_ADDR=0.0.0.0
-OP_NODE_METRICS_ENABLED=true
-OP_NODE_METRICS_PORT=7300
-OP_NODE_NETWORK=base-mainnet
+BASE_NODE_L2_ENGINE_RPC=ws://execution:8551
+BASE_NODE_L2_ENGINE_AUTH=/tmp/engine-auth-jwt
+BASE_NODE_L2_ENGINE_AUTH_RAW=688f5d737bad920bdfb2fc2f488d6b6209eebda1dae949a8de91398d932c517a
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BASE_NODE_L2_ENGINE_AUTH_RAW key should go before the BASE_NODE_L2_ENGINE_RPC key

(UnorderedKey)